manifests/a/Autodesk,Inc/AutodeskDownloadManager/4.0.19.0/Autodesk,Inc.AutodeskDownloadManager.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Autodesk,Inc.AutodeskDownloadManager
+PackageVersion: 4.0.19.0
+Installers:
+- Architecture: x64
+  InstallerType: exe

```

Comment:
Package does not install silently with the current Installer Type; Silent installation switches must be provided